src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseService.java

```diff
@@ -76,14 +76,27 @@ public void save(
             updateDirectionsQuestionnaireDeadline(updatedClaim, authorization);
         }
         Response response = claim.getResponse().orElseThrow(IllegalArgumentException::new);
-        if (!isReferredToJudge(claimantResponse)
-            || (isReferredToJudge(claimantResponse) && PartyUtils.isCompanyOrOrganisation(response.getDefendant()))) {
+        if (!isSettlementAgreement(claim, claimantResponse)
```

Comment:
Do we have a test case on situation where it is settlement making sure default event is not invoked?

Reply:
Once test case is provided, I'm happy to approve.